src/service.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import './polyfills';
```

Comment:
We really need a better solution. Basically you need to put it everywhere that uses promises and has an immediate side effect.